Alderis/UIColor+HBAdditions.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  UIColor+HBAdditions.swift
+//  Alderis
+//
+//  Created by Ryan Nair on 10/5/20.
+//  Copyright © 2020 HASHBANG Productions. All rights reserved.
+//
+
+import UIKit
+
+extension UIColor {
+	
+	/// Initializes and returns a color object using data from the specified object.
+	///
+	/// The value is expected to be one of:
+	///
+	/// * An array of 3 or 4 integer RGB or RGBA color components, with values between 0 and 255 (e.g.
+	///   `@[ 218, 192, 222 ]`)
+	/// * A CSS-style hex string, with an optional alpha component (e.g. `#DAC0DE` or `#DACODE55`)
+	/// * A short CSS-style hex string, with an optional alpha component (e.g. `#DC0` or `#DC05`)
+	///
+	/// @param value The object to retrieve data from. See the discussion for the supported object
+	/// types.
+	/// @returns An initialized color object. The color information represented by this object is in the
+	/// device RGB colorspace.
+	
+	@objc convenience init?(hbcp_propertyListValue value: Any?) {
+		
+			if value == nil {
+						return nil
+			} else if value is Array<Any> && (value as! Array<Any>).count == 3 || (value as! Array<Any>).count == 4 {
+						let array = value as! [CGFloat]
+						self.init(
+								red: array[0] / 255.0,
+								green: array[1] / 255.0,
+								blue: array[2] / 255.0,
+								alpha: CGFloat(array.count == 4 ? (array[3]) : 1))
```

Comment:
Code style: remove unneeded brackets here. Also I think `CGFloat(…)` may be unnecessary, it should infer that from the cast to `[CGFloat]` above.